pytorch_lightning/core/lightning.py

```diff
@@ -893,10 +893,6 @@ def init_ddp_connection(self, global_rank: int, world_size: int, is_slurm_managi
         log.info(f"initializing ddp: GLOBAL_RANK: {global_rank}, MEMBER: {global_rank+1}/{world_size}")
         torch_distrib.init_process_group(torch_backend, rank=global_rank, world_size=world_size)
 
-    """
```

Comment:
why did we remove this?

Reply:
It looks like it does not belong here. My question would be why is it here in the first place?

Reply:
@ananyahjha93 ?